domain/src/main/java/org/oppia/domain/util/InteractionObjectExtensions.kt

```diff
@@ -36,7 +39,8 @@ private fun NumberWithUnits.toAnswerString(): String {
   val suffixedUnits = unitList.filterNot(::isPrefixUnit)
 
   val prefixString = prefixedUnits.joinToString(separator = " ")
-  val suffixedString = suffixedUnits.joinToString(separator = " ", transform = NumberUnit::toAnswerStringPart)
+  val suffixedString =
+    suffixedUnits.joinToString(separator = " ", transform = NumberUnit::toAnswerStringPart)
```

Comment:
I couldn't see what has changed here, is it just formatting? If so, have you run the kotlin lint check on this?

Reply:
Yeah for this kind of thing can we add lint check on pre commit hooks just like oppe web so that whosoever is reviewing don't waste hir/her time in checking linting issues

Reply:
Yeah we do have one , its not on pre commit right now but its on our continuous integration. But it looks like you are working on the domain module and we haven't enabled that yet. One easy way to do it yourself is to install ktlint and run ktlint -f <path to your file> 
It will format whatever it can and throw errors whenever it cant

Reply:
okay thanks will do 👍